apis/kueue/v1beta1/workload_types.go

```diff
@@ -296,6 +314,15 @@ type WorkloadList struct {
 	Items           []Workload `json:"items"`
 }
 
+// QueueingPolicyAction specifies if a suspended job should be queued.
+type QueueingPolicyAction string
```

Comment:
Just `QueueingPolicy` or `QueueingPolicyType` is fine since this is not an action